implementations/rust/ockam/ockam_macros/src/node_test_attribute/parser.rs

```diff
@@ -50,9 +49,8 @@ fn output_node_test(
     input.block = parse2(quote_spanned! {last_stmt_end_span=>
         {
             use core::time::Duration;
-            use ockam_node::tokio::time::timeout;
 
-            let (mut #ctx_ident, mut executor) = ockam_node::start_node();
```

Comment:
This is the main change to the macro.

Reply:
Hm, so now it has to be explicitly imported? I don't think this is a great experience, to be honest, I suspect the first time use of this macro would always include a hard to understand error.

Another option here would be to accept a `#[ockam::node(crate = "ockam_node")]` or `#[ockam::node(crate = "ockam")]`[^1], or `#[ockam::node(crate = "crate")]`, similar to https://serde.rs/container-attrs.html#crate. If not provided, I think it could default to `ockam_node`.

So, I think I'd prefer this, if possible.

[^1]: I think we'd have to move the declaration of `start_node` to somewhere else for this to actually work, admittedly.

Reply:
I have some changes ready but there are some details I wanted to clarify first:

1. The default crate shouldn't be `ockam` instead of `ockam_node`? If the entry point of the public API is the `ockam` crate, then every time a user wanted to write a test would have to set the `crate` attribute in the macro. Most of our internal tests will have to be modified to set the `crate` attribute though.
2. I've been researching about not using a string in the attribute and I'm pretty much sure there is no other way to do it. Do you have any examples of this in mind? The thing is that an attribute is a `MetaNameValue` and the value is a `Lit` which apparently can only be a string, an integer or a boolean (see https://docs.rs/syn/latest/syn/struct.MetaNameValue.html)

Reply:
As discussed internally:

1. We want to default to `ockam_node`
2. We will consider modifying how the `crate` attribute is parsed using the [`MetaList`](https://docs.rs/syn/latest/syn/struct.MetaList.html) type, which would allow us to write the value as a path instead of as a string (e.g. `#[ockam::test(crate = path::to::crate)]`